ecosystem/lava-sdk/src/stateTracker/stateQuery/state_chain_query.ts

```diff
@@ -187,6 +235,70 @@ export class StateChainQuery {
     }
   }
 
+  public async updateDowntimeParams(): Promise<
```

Comment:
we cannot add more queries on SDK initialization. this slows down our SDK, which is the reason we created the SDK query. if downtime params are required for downtime handling on the SDK we can add it to the SDK query.